schema/swagger-extensions.json

```diff
@@ -1122,6 +1142,9 @@
         "example": { },
         "x-ms-client-flatten": {
           "$ref": "#/definitions/xmsClientFlatten"
+        },
+        "x-ms-client-name": {
+          "type": "string"
```

Comment:
I took it out because referring to 'parameter' was wrong and I didn't think of the right term to use here -- 'element,' 'property,' etc.